### PR TITLE
Pills update to new IA

### PR DESCRIPTION
**Pull request by chri2547**
@palindromebart , pills is ready for final review and merge. Editorial feedback has been incorporated.

image

### CODE REVIEW

File: _component_design/pills.md [hunk 1]
```diff
@@ -71,32 +139,60 @@ A pill is different than a tag in a variety of ways, with the most important bei
...
-## Specifications
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg"" %}
+
+## States
+
+Pills can exist in active, hover, too many, or too long states.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/pills.md]: done.

File: _component_design/pills.md [hunk 2]
```diff
@@ -51,18 +117,20 @@ When using pills, ensure they are uniform in size, shape, and application. Pills
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Default layout
+### Spacing
+
+Place 16px of horizontal space and 4px of vertical space between a pill an another component. Use the best top spacing for your particular use case.
 
-Pills are a component with a variety of use cases, so they can be used for many ways of indicating something has been selected: keyword search, input text search, the name of a server you’ve selected. If it can be selected, you can make a pill of it.
+Unless you have a reason, justify pills to the left.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/pills.md]: done.

File: _component_design/pills.md [hunk 3]
```diff
@@ -5,38 +5,104 @@ secondary: Editing
 layout: component
 category: Components
 usage: >
-  Pills are small, pill-shaped containers that indicate selection. They are useful for their small size and for the ease with which they can be removed. As selection indicators (not selectors themselves), they have a broad set of possible applications.
+  A pill is a small, oval-shaped component that indicates selection. Pills are often associated with tags, however, you can also apply pills to keyword search terms, input text search, or the name of a server selected by the user. You can represent any selectable object with a pill.
```
[the remark this answers is not in the document]
Comment (palindromebart) [reply _component_design/pills.md]: Ok, I spoke with drew and ryan. 
we agreed to cut any reference to tags.

replace the above text with

"A pill is a small, dismissible, oval-shaped component that indicates selection. Pills are used to provide feedback of selected keyword search terms, input text search, or the name of a server selected by the user. You can represent any selectable object with a pill.

File: _component_design/pills.md [hunk 4]
```diff
@@ -5,38 +5,104 @@ secondary: Editing
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+When using pills, ensure they are uniform in size, shape, and application.
```
[the remark this answers is not in the document]
Comment (palindromebart) [reply _component_design/pills.md]: remove "and application"

**Comment by cyrichardson** [comment]
On it, @chri2547.

**Comment by cyrichardson** [comment]
@chri2547 Suggested changes here: 
[Helix-pills-section_cyrEdits.docx](https://github.com/rackerlabs/design-system/files/2032316/Helix-pills-section_cyrEdits.docx)

**Pull request merged**